app/api/data-fetching/routes.ts

```diff
@@ -0,0 +1,51 @@
+import { IAPINotRespondingError } from '#models/api-not-responding';
+import { getCarteProfessionnelleTravauxPublic } from '#models/espace-agent/carte-professionnelle-travaux-publics';
+import { EScope } from '#models/user/rights';
+
+const NotImplementedHandler = () => {
+  throw new Error('Not implemented');
+};
+
+export const APIHandler = {
+  'espace-agent/carte-professionnelle-TP': getCarteProfessionnelleTravauxPublic,
+  'espace-agent/conformite': NotImplementedHandler,
+  'espace-agent/opqibi': NotImplementedHandler,
+  'espace-agent/qualibat': NotImplementedHandler,
+  'espace-agent/qualifelec': NotImplementedHandler,
+  'espace-agent/rcs-mandataires': NotImplementedHandler,
+  'espace-agent/rne/documents': NotImplementedHandler,
```

Comment:
If we do so, it will lead to some network error during the deployment, as we'll change the route of the resource. It's not that much of deal, I think we can be ok with that.

I'm not a big fan of Enum. While they were necessary during the first days of Typescript, we can now have the same type safety power with `as const` without the need to add another specific data structure (and adding one level of indirection over string). 

Their main justification is not to improve typing, but to allow for smoother transition if we decide to rename a route.